android/app/src/main/java/com/mattermost/share/ShareModule.java

```diff
@@ -40,6 +40,7 @@
     public ShareModule(ReactApplicationContext reactContext) {
         super(reactContext);
     }
+    private String tempFolder;
```

Comment:
Worth using a `Path` here instead of having to juggle `/` on a `String`?

Reply:
Will look at making this a Path